Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.4.10 (unreleased)
 
 Features:
+ * Add ``assert(condition)`` to abort execution.
```

Comment:
This is a bit too sloppy (people might stop reading here). What about:

    Add ``assert(condition)``, which throws if condition is false.